docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -365,34 +385,40 @@ You can modify the flex item placement through the following properties:
 
     To create the item alignment shown in the above figure:
 
-    ```
+    ```cs
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
 
-    /// Lay out the items horizontally
-    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
-
     /// Set the default alignment along the cross axis to the beginning of the container
     flexContainer.AlignItems = FlexContainer.Alignment.AlignFlexStart;
+    flexContainer.WidthResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.HeightResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
+    Window.Instance.GetDefaultLayer().Add(flexContainer);
 
     /// Create flex items and add them to the flex container
     View item1 = new View();
+
     /// Align item1 to the center of the container
-    item1.AlignSelf = FlexContainer.Alignment.AlignCenter;
+    item1.AlignSelf = (int)FlexContainer.Alignment.AlignCenter;
+    item1.Size2D = new Size2D(200, 200);
+    item1.BackgroundColor = Color.Red;
     flexContainer.Add(item1);
 
     View item2 = new View();
+
     /// Align item2 to the beginning of the container based on the default alignment
+    item2.Size2D = new Size2D(200, 200);
+    item2.BackgroundColor = Color.Green;
     flexContainer.Add(item2);
```

Comment:
Question. According to the "above figure", the second item should has "flexStart" property. Don't we need to set "flexStart" to item2?

Reply:
As it is said in comment: 
> Align item2 to the beginning of the container based on the default alignment

Lack of `AlignSelf` is intended to show alignment inheritance